### PR TITLE
refactor: purge all quay images

### PR DESCRIPTION
**Pull request by devnote-dev**
# Description

This PR updates all eggs using the old `quay.io` images to the `ghcr.io` images as well as switches them to the v2 format. Some eggs will be excluded from this as they already have PRs which will resolve this.

## Eggs List

bots:
* discord
* * [x] altbot
* * [x] bastion
* * [x] coprbot
* * [x] dynamic
* * [x] fragbot
* * [x] jmusicbot
* * [x] parkertron
* * [x] pixelbot
* [ ] other/bigbrotherbot
* [ ] twitch/sogebot

databases/sql:
* [ ] MariaDB
* [ ] PostgreSQL

game_eggs:
* [ ] FoundryVTT
* among_us
* * [ ] crewlink-server
* * [ ] impostor-server
* [ ] beamng/kissmp
* [ ] classicube/mcgalaxy
* [ ] enemy_territory/etlegacy
* factorio
* * [ ] factorio-modupdate
* * [ ] factorio
* [ ] ftl/tachyon
* gta
* * [ ] gtac
* * [ ] mtasa
* * [ ] ragemp
* * [ ] samp
* [ ] losanglescrimes
* [ ] minetest
* [ ] mohaa
* [ ] openarena
* openra
* * [ ] openra_dune2000
* * [ ] openra_redalert
* * [ ] openra_tiberian_dawn
* [ ] starmade
* steamcmd_servers
* * [ ] 7_ways_to_die
* * [ ] ark_survival_evolved
* * arma
* * * [ ] arma3
* * * [ ] arma_reforger
* * assetto_corsa
* * [ ] avorion
* * [ ] barotrauma
* * [ ] citadel
* * [ ] cryofal
* * [ ] dayz-experimental
* * [ ] dont_starve
* * [ ] eco
* * [ ] fof
* * [ ] holdfast
* * [ ] hurtworld
* * [ ] insurgency_sandstorm
* * [ ] killing_floor_2
* * [ ] left4dead
* * [ ] left4dead2
* * [ ] modiverse
* * [ ] mordhau
* * [ ] nmrih
* * [ ] onset
* * [ ] pavlov_vr
* * [ ] pixark
* * [ ] post_scriptum
* * [ ] project_zomboid
* * [ ] quake_live
* * [ ] risk_of_rain_2
* * scpsl
* * * [ ] dedicated
* * * [ ] exiled
* * * [ ] multiadmin
* * [ ] soldat
* * [ ] squad
* * [ ] stationeers
* * [ ] subnautica_nitrox_mod
* * [ ] svencoop
* * [ ] tower_unite
* * [ ] unturned
* [ ] teeworlds
* [ ] tycoon_games/openttd
* [ ] vintage_story
* [ ] wine/generic
* [ ] xonotic

generics:
* [ ] deno
* [ ] lua/luvit
* [ ] nodemon

software:
* [ ] code-server
* [ ] elasticsearch
* [ ] gitea
* [ ] grafana
* [ ] languagetool
* [ ] owncast
* [ ] rabbitmq
* [ ] reposilite

voice_servers:
* [ ] lavalink
* [ ] teamspeak
* [ ] ts3_manager

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### CODE REVIEW

File: bots/discord/atlbot/egg-a-t-l-bot.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
     },
     "exported_at": "2019-06-23T10:00:12-04:00",
     "name": "ATLBot",
     "author": "jfeldt19@gmail.com",
     "description": "ATLbot in ptero\r\n\r\nhttps:\/\/github.com\/ATLauncher\/discord-bot\/",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:alpine_nodejs-10",
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:alpine": "ghcr.io\/parkervcp\/yolks:alpine"
```
Comment (parkervcp) [inline bots/discord/atlbot/egg-a-t-l-bot.json:11]: This was specifically nodejs

File: bots/discord/parkertron/egg-parkertron.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
     },
     "exported_at": "2020-04-03T12:50:58-04:00",
     "name": "parkertron",
     "author": "parker@parkervcp.com",
     "description": "The stupid chatbot parkertron by Parkervcp.\r\n\r\nhttps:\/\/github.com\/parkervcp\/parkertron",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:bot_parkertron",
+    "docker_images": {
+        "parkertron": "ghcr.io\/parkervcp\/yolks:bot_parkertron"
```
Comment (parkervcp) [inline bots/discord/parkertron/egg-parkertron.json:11]: Don't touch this one. I am working on it already.

File: bots/discord/pixelbot/egg-pixel-bot.json [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
     },
     "exported_at": "2020-01-19T11:11:24-05:00",
     "name": "pixel-bot",
     "author": "parker@parkervcp.com",
     "description": "A Discord bot written in Python using discord.py\r\n\r\nhttps:\/\/github.com\/Ispira\/pixel-bot",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:alpine_python-3",
```
Comment (parkervcp) [inline bots/discord/pixelbot/egg-pixel-bot.json:10]: pretty sure this is just a single image

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: bots/discord/pixelbot/egg-pixel-bot.json (hunk 3 above)
Comment (devnote-dev) [reply bots/discord/pixelbot/egg-pixel-bot.json:10]: The egg only really mentions 3.6/3.7 so I was unsure, `~>3.6` versions should work with it anyway but I've removed them for now.

File: bots/discord/atlbot/egg-a-t-l-bot.json (hunk 1 above)
Comment (devnote-dev) [reply bots/discord/atlbot/egg-a-t-l-bot.json]: Yep, updated it to NodeJS 16 as that's their minimum supported version now.

**Comment by gOOvER** [comment]
pls also think about, to change the Install Images for Steam Games to the Installer Image

**Pull request closed without merging**